src/platform/unittest/DataPathTest.cpp

```diff
@@ -429,10 +455,12 @@ TEST_P(DataPathTest, DataRebind)
 
     QuicEventInitialize(&RecvContext.ClientCompletion, FALSE, FALSE);
 
+    QUIC_ECN_TYPE ecn = std::get<1>(GetParam());
```

Comment:
I'd prefer not to reuse the DataRebind test, and just make a new one.

Reply:
Ok. I will get to this next week.